### PR TITLE
[RDX-272] Enable CRT for release/0.3.x branch

### PR DESCRIPTION
**Pull request by claire-labry**
This PR enables CRT for the `release/0.3.x` branch. 

Tests are passing [here.](https://github.com/hashicorp/crt-workflows-common/actions/runs/1775487996)

### CODE REVIEW

File: .release/docker/docker-entrypoint.sh [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+CTS_CONFIG_DIR=/consul-terraform-sync/config
+
+# Set the configuration directory
+if [ "$1" = '/bin/consul-terraform-sync' ]; then
+	shift
+	set -- /bin/consul-terraform-sync -config-dir="$CTS_CONFIG_DIR" "$@"
+fi
+
+exec "$@"
+© 2022 GitHub, Inc.
```
Comment (sarahethompson) [inline .release/docker/docker-entrypoint.sh:33]: It looks like these 2 lines at the bottom of the file have sneaked in here somehow!

**Review by sarahethompson — APPROVED** [review]
LGTM @claire-labry! I love the improvements to simplify `build.yml` 🎉 
I'd maybe push up a PR to `main` and `release/0.4.x` with the build.yml changes too, just to keep all the branches in sync and to help with future backporting.
I see there's a slight difference with the name of `consul-terraform-sync-0.2.2.aarch64.rpm`  vs `consul-terraform-sync-0.5.0.arm64.rpm` (on the builds from `main` and `release/0.4.x`) due to the new `Set Package Names` job. this is probably fine but might be worth checking with Kyle!

**Review by lornasong — APPROVED** [review]
looks good from my side of things! thanks for updating our release branches 💞.  feel free to merge whenever you're all set

File: .release/docker/docker-entrypoint.sh (hunk 1 above)
Comment (claire-labry) [reply .release/docker/docker-entrypoint.sh:33]: whoops, great eye! will fix that.

**Comment by claire-labry** [comment]
Great call out @sarahethompson! I'll update both `main` and `release/0.4.x` branches. I can def check-in with Kyle 😁

**Comment by claire-labry** [comment]
For posterity's sake, I chatted with Kyle and he mentioned that `main` should be updated to match `consul-terraform-sync-0.2.2.aarch64.rpm` formatting :)

**Pull request merged**